src/ModelContextProtocol/Protocol/Transport/SseResponseStreamTransport.cs

```diff
@@ -76,7 +76,8 @@ public async Task SendMessageAsync(IJsonRpcMessage message, CancellationToken ca
             throw new InvalidOperationException($"Transport is not connected. Make sure to call {nameof(RunAsync)} first.");
         }
 
-        await _outgoingSseChannel.Writer.WriteAsync(new SseItem<IJsonRpcMessage?>(message), cancellationToken);
+        // Emit redundant "event: message" lines for better compatibility with other SDKs.
+        await _outgoingSseChannel.Writer.WriteAsync(new SseItem<IJsonRpcMessage?>(message, "message"), cancellationToken);
```

Comment:
Instead of `"message"`, you could also use `SseParser.EventTypeDefault`.

Reply:
But `"message"` is so much shorter! 😆 I changed it. It does communicate that "message" is the default to those less familiar with SSE (although omitting the event is the *default* default 😝), and I already have "message" in the comment a line above.